### DISCUSSION TITLE
EZlaunch tutorial

### DISCUSSION THREAD

**Discussion by banns46** [aptos-labs/aptos-developer-discussions#283] [Questions] [answered]
### Describe your question in detail.

Hello, I tried this tutorial https://learn.aptoslabs.com/example/nft-launchpad-guide/quick-setup and it worked great for me on testnet, is there mainnet address for ez_launch if I want to launch my collection on mainnet?

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by jillxuu** [comment, marked as answer]
Hi @banns46 i'm glad you liked this tutorial on testnet. We launch this contract to mainnet at this address `0xde47c6b9d846c03b928d7707df253ffad4ce911af56fbe26af218fd544adcbec`. i'll soon update the tutorial to reflect this as well. thanks!
